index.html

```diff
@@ -1,168 +1,309 @@
+
+      <li class="box three-col">
+          <a href="http://pellaeon.github.io/bsd-cloudinit">
+              <div>
+                  <img src="https://assets.ubuntu.com/v1/6fa4b259-freebsd.png" alt="FreeBSD logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/23835286-partner-logo-rti.png" alt="Real-Time Innovations">
+
+      <li class="box three-col">
+          <a href="https://getfedora.org/en/cloud/download/">
+              <div>
+                  <img src="https://assets.ubuntu.com/v1/0be4c6c4-fedora.png" alt="Fedora logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/e6bc8a26-partner-logo-osrf.png" alt="OSRF">
+
+      <li class="box three-col">
+          <a href="https://packages.gentoo.org/package/app-emulation/cloud-init">
+              <div>
+                  <img src="https://assets.ubuntu.com/v1/cb8e3497-gentoo.png" alt="Gentoo logo">
+              </div>
+          </a>
       </li>
-      <li class="inline-logos__item">
-          <img class="inline-logos__image" src="https://assets.ubuntu.com/v1/d27eee64-partner-logo-banana-pro.png" alt="LeMaker">
+
+      <li class="box three-col last-col">
+          <a href="https://susestudio.com/">
+              <div>
+                  <img src="https://assets.ubuntu.com/v1/2331fa6f-open-suse.png" alt="OpenSUSE logo">
+              </div>
+          </a>
       </li>
-    </ul>
-  </div>
-</section>
 
-<!-- Source Code -->
-<section class="row">
-  <h2>Source Code</h2>
+      <!-- <li class="box three-col"> QUERY: should this stay?
+          <a href="http://download.cirros-cloud.net/">
+              <div>
+                  <img src="https://assets.ubuntu.com/v1/c1e30825-cirros.png" alt="cirrOS logo">
+              </div>
+          </a>
+      </li> -->
 
-  <div class="four-col">
-    <div>Browser and branch from Bazaar</div>
-  </div>
+      </ul>
 
-  <div class="four-col">
-    <div>Browse, branch, fork or clone from Github</div>
-  </div>
 
-  <div class="four-col">
-    <div>Download the offically released archives</div>
-  </div>
-</section>
 
-<!-- Public Cloud Examples -->
-<section class="row">
-  <div class="twelve-col">
-    <h2>Used across the public cloud</h2>
-    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. In qui adipisci
-      assumenda, enim ipsum consequatur exercitationem, temporibus laudantium
-      beatae unde ut molestiae, corporis? Quos adipisci, provident, vel sed non.
-    </p>
+  </section>
 
-  </div>
-</section>
+  <!-- Source Code -->
+  <section class="row source-code">
+    <h2>Source Code</h2>
 
-<!-- Support channels -->
-<section class="row">
+    <div class="flex-wrap">
 
-  <div class="twelve-col">
+      <div class="four-col">
+        <div class="box">
```

Comment:
@anthonydillon This markup structure is required to keep the three boxes an equal height with Flexbox.